wiiu/wiiuMainApplication/main-application.ts

```diff
@@ -1,10 +1,10 @@
 export default interface WiiUMainApplicationAPI {
 	/**
-	 * Unknown
-	 * @param unk - Unknown. Maybe a screen ID?
-	 * @returns Unknown
+	 * Returns a JPEG base64 screenshot
```

Comment:
Slight reword for clarity

```suggestion
	 * Returns a base64 encoded JPEG screenshot of the selected screen
```